lua/telescope/builtin/git.lua

```diff
@@ -137,6 +137,26 @@ git.status = function(opts)
   }):find()
 end
 
+git.recents = function(opts)
+  local depth = utils.get_default(opts.depth, 5)
+
+  if opts.cwd then
+    opts.cwd = vim.fn.expand(opts.cwd)
+  end
```

Comment:
You don't need to handle `cwd`. When exporting the function we call `apply_checks` which will handle `cwd`